style.css

```diff
@@ -0,0 +1,188 @@
+body { font-family: "3rd party font", serif; }
+
+.Mobile_background {
+  width: 365px;
+  height: 5410px;
+  left: 0;
+  background: #f7f7f9;
+}
+
+header {
+  position: static;
+  width: 375px;
+  height: 92px;
+  left: 0;
+  top: 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  position: static;
```

Comment:
- [ ] [Optional] I strongly suggest you do not modify default values for HTML elements unless necessary, this will help you to avoid unexpected bugs and undesired behavior. This also will shorten the code, so you only write the required code, nothing more or less